CHANGES/5050.feature

```diff
@@ -0,0 +1,2 @@
+A new Collection uploader has been added to the pulp_ansible API at
+``/pulp/api/v3/ansible/collection/``.
```

Comment:
nitpick - needs to be collections

Reply:
that's not a nitpick that's just correctness :)